ui/src/app/components/gke-app-configuration-panel.spec.tsx

```diff
@@ -51,8 +56,10 @@ const validateInitialLoadingSpinner = async () => {
 
 describe(GKEAppConfigurationPanel.name, () => {
   beforeEach(() => {
+    jest.clearAllTimers();
```

Comment:
I made these 2 changes to cope with `maybeStartPollingForUserApps()`
because the tests were interfering with each other after this change